main/menu.js

```diff
@@ -17,12 +17,32 @@ var dialog = require('./dialog')
 var shell = require('./shell')
 var windows = require('./windows')
 var thumbnail = require('./thumbnail')
+var State = require('../renderer/lib/state')
 
-var menu
+var menu, state
 
 function init () {
   menu = electron.Menu.buildFromTemplate(getMenuTemplate())
   electron.Menu.setApplicationMenu(menu)
+
+  State.load(onState)
```

Comment:
We def don't want to load `config.json` in two different places. 

Does "Open in VLC" have to be a configuration pref? What if we just added it to the right-click menu in the torrent list